python/aida_interchange/aifutils.py

```diff
@@ -391,17 +391,17 @@ def mark_as_mutually_exclusive(g, alternatives, system, none_of_the_above_prob):
 
     mutual_exclusion_assertion = _make_aif_resource(g, None, AIDA_ANNOTATION.MutualExclusion, system)
 
-    for alts in alternatives:
+    for (edges_for_alternative, confidence) in alternatives.items():
```

Comment:
@rgabbard It looks like this function is now expecting "alternatives" to be a dict. I made changes to the Examples.py tests that invoke this function. Are the changes that I made similar to how you're calling this function?